presto-spi/src/main/java/com/facebook/presto/spi/function/JavaAggregationFunctionImplementation.java

```diff
@@ -34,6 +34,8 @@
 
     boolean isOrderSensitive();
 
+    boolean isCalledOnNullInput();
```

Comment:
i don't think this needs to be in the aggregation function implementation.  it should be enough that it will be in the function metadata.  It doesn't look like it's used anywhere.

Reply:
I was going to use this to determine whether the agg function should participate in the rewrite in the other PR. But I see your point - I will just override the existing method in BuiltInFunction